### PR TITLE
a handy script for running experiments in one command line

### PR DESCRIPTION
**Pull request by le-horizon**
This is a wrapper script which allows running experiments very easy, e.g.
`run accartpole-env_4` will run cart pole using 4 parallel envs.
For playing, run
`run accartpole-env_4 --play`

### CODE REVIEW

File: alf/bin/run.py [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+```bash
+cd ${PROJECT}/alf/examples;
+python -m alf.bin.run \
+  actargetnav-randgoal_False-goalname_ball_car^wheel-env_30-image_84_84-unroll_100-msteps_100-mepisteps_1000-faildist_5-rnn_True-lstm_128_128 \  # first parameter is the run tag, which will be parsed
+  --cluster=None \
+  --path=~/tmp \  # path to store output logs
+  --play
+```
+
+Use '=' to replace '-', and '^' to replace '_' in argument values.
```
Comment (emailweixu) [inline alf/bin/run.py]: What does this mean?

File: alf/bin/run.py [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+import re
+import sys
+
+from absl import app
+from absl import flags
+from absl import logging
+
+flags.DEFINE_string("path", os.path.join(os.path.expanduser("~"), "tmp"),
+                    "Root directory for all runs and all root_dir's.")
+flags.DEFINE_string('cluster', None, 'idc, us or None for local.')
```
Comment (emailweixu) [inline alf/bin/run.py]: what is the effect of this?

File: alf/bin/run.py [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""Run training and playing in a simple one liner command.
+Allows running on clusters or locally.
+
+To run actor_critic on gym CartPole:
+```bash
+cd ${PROJECT}/alf/examples;
+python -m alf.bin.run \
+  actargetnav-randgoal_False-goalname_ball_car^wheel-env_30-image_84_84-unroll_100-msteps_100-mepisteps_1000-faildist_5-rnn_True-lstm_128_128 \  # first parameter is the run tag, which will be parsed
```
Comment (emailweixu) [inline alf/bin/run.py]: need to explain what this long string mean.

File: alf/bin/run.py [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+
+    load_fn = get_env_load_fn(gin_file)
+
+    args = gen_args(named_tags, task, load_fn)
+
+    if os.path.exists("../bin/train.py"):
+        module = train
+        additional_flag = ""
+    else:
+        module = "main"
```
Comment (emailweixu) [inline alf/bin/run.py]: where is module "main" from?

File: alf/bin/run.py (hunk 2 above)
Comment (emailweixu) [reply alf/bin/run.py]: ALF_GIN_MAP_FILE=... command gin_file A=B C=D [root_prefix]

**Comment by le-horizon** [comment]
Make sense not to pollute the repo if I am the only person using this.  Closing this pull request, but keeping the branch.  If anyone else is interested in using the script, let me know.

**Pull request closed without merging**